lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -572,82 +576,88 @@ void SwiftREPL::CompleteCode(const std::string &current_code,
             swift_ts->GetSwiftASTContext(nullptr));
     m_swift_ast = target_swift_ast;
   }
-  SwiftASTContextForExpressions *swift_ast = m_swift_ast;
-
-  if (swift_ast) {
-    ThreadSafeASTContext ast = swift_ast->GetASTContext();
-    swift::REPLCompletions completions;
-    SourceModule completion_module_info;
-    completion_module_info.path.push_back(ConstString("repl"));
-    swift::ModuleDecl *repl_module = nullptr;
-    if (m_completion_module_initialized)
-      repl_module = swift_ast->GetModule(completion_module_info, error);
-    if (repl_module == nullptr) {
-      swift::ImplicitImportInfo importInfo;
-      importInfo.StdlibKind = swift::ImplicitStdlibKind::Stdlib;
-      repl_module = swift_ast->CreateModule(completion_module_info, error,
-                                            importInfo);
-      auto bufferID = (*ast)->SourceMgr.addMemBufferCopy("// swift repl\n");
-      swift::SourceFile *repl_source_file = new (**ast) swift::SourceFile(
-          *repl_module, swift::SourceFileKind::Main, bufferID);
-      repl_module->addFile(*repl_source_file);
-      swift::performImportResolution(*repl_source_file);
-      m_completion_module_initialized = true;
+  return m_swift_ast;
+}
+
+void SwiftREPL::CompleteCode(const std::string &current_code,
+                             CompletionRequest &request) {
+  auto *swift_ast = getSwiftASTContext();
+  if (!swift_ast)
+    return;
+
+  Status error;
+  ThreadSafeASTContext ast = swift_ast->GetASTContext();
+  swift::REPLCompletions completions;
+  SourceModule completion_module_info;
+  completion_module_info.path.push_back(ConstString("repl"));
+  swift::ModuleDecl *repl_module = nullptr;
+  if (m_completion_module_initialized)
+    repl_module = swift_ast->GetModule(completion_module_info, error);
+  if (repl_module == nullptr) {
```

Comment:
```suggestion
  if (!repl_module) {
```

Reply:
I should note this is code that was already there, but yeah good catch, thanks!